[2025-04-09T21:37:11.882+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline_modo_prueba.load_to_sqlite manual__2025-04-09T21:23:26.008365+00:00 [queued]>
[2025-04-09T21:37:11.894+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline_modo_prueba.load_to_sqlite manual__2025-04-09T21:23:26.008365+00:00 [queued]>
[2025-04-09T21:37:11.894+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-09T21:37:11.939+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): load_to_sqlite> on 2025-04-09 21:23:26.008365+00:00
[2025-04-09T21:37:11.946+0000] {standard_task_runner.py:60} INFO - Started process 258 to run task
[2025-04-09T21:37:11.951+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline_modo_prueba', 'load_to_sqlite', 'manual__2025-04-09T21:23:26.008365+00:00', '--job-id', '206', '--raw', '--subdir', 'DAGS_FOLDER/etl_workflow.py', '--cfg-path', '/tmp/tmptg83wxuo']
[2025-04-09T21:37:11.953+0000] {standard_task_runner.py:88} INFO - Job 206: Subtask load_to_sqlite
[2025-04-09T21:37:12.041+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_pipeline_modo_prueba.load_to_sqlite manual__2025-04-09T21:23:26.008365+00:00 [running]> on host 705bf0793dae
[2025-04-09T21:37:12.189+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='simon' AIRFLOW_CTX_DAG_ID='etl_pipeline_modo_prueba' AIRFLOW_CTX_TASK_ID='load_to_sqlite' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T21:23:26.008365+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-09T21:23:26.008365+00:00'
[2025-04-09T21:37:12.190+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-09T21:37:12.191+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/***/scripts/load_to_sqlite.py']
[2025-04-09T21:37:12.205+0000] {subprocess.py:86} INFO - Output:
[2025-04-09T21:37:13.796+0000] {subprocess.py:93} INFO - âœ… Archivo parquet leÃ­do: /opt/***/data/processed/final_merged.parquet
[2025-04-09T21:37:13.796+0000] {subprocess.py:93} INFO - ðŸ”— Conectado a la base de datos SQLite en /opt/***/data/output/merged_data.db
[2025-04-09T21:37:13.797+0000] {subprocess.py:93} INFO - ðŸ“¦ Datos cargados en la tabla 'datos_finales'
[2025-04-09T21:37:13.797+0000] {subprocess.py:93} INFO - âœ… ConexiÃ³n cerrada. Proceso de carga completado.
[2025-04-09T21:37:13.906+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-09T21:37:13.966+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_pipeline_modo_prueba, task_id=load_to_sqlite, execution_date=20250409T212326, start_date=20250409T213711, end_date=20250409T213713
[2025-04-09T21:37:14.054+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-09T21:37:14.082+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
